### PR TITLE
Support for mouse wheel EventListener

### PR DESCRIPTION
**Pull request by mattthw**
+ added new MouseWheelMovedCanvasEvent
+ added new CanvasMouseWheelListener
+ added mouse wheel listener to gameAPI's canvas

### CODE REVIEW

File: src/main/java/com/wingman/client/rs/listeners/CanvasMouseWheelListener.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.wingman.client.rs.listeners;
+
+import com.wingman.client.api.events.mouse.MouseWheelMovedCanvasEvent;
+import com.wingman.client.plugin.PluginManager;
+
+import java.awt.event.MouseWheelEvent;
+import java.awt.event.MouseWheelListener;
+
+/**
+ * Created by mmccoy37 on 1/18/17.
```
Comment (alexanderhenne) [inline src/main/java/com/wingman/client/rs/listeners/CanvasMouseWheelListener.java]: Please remove this comment

File: src/main/java/com/wingman/client/api/events/mouse/MouseWheelMovedCanvasEvent.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import com.wingman.client.api.event.EventListener;
+import com.wingman.client.api.event.EventListenerList;
+
+import java.awt.event.MouseWheelEvent;
+
+public class MouseWheelMovedCanvasEvent extends Event {
+
+    public static final EventListenerList eventListenerList = new EventListenerList();
+
+    MouseWheelEvent mouseWheelEvent;
```
Comment (alexanderhenne) [inline src/main/java/com/wingman/client/api/events/mouse/MouseWheelMovedCanvasEvent.java]: Make this field ```public``` and add a newline under it

File: src/main/java/com/wingman/client/api/events/mouse/MouseWheelMovedCanvasEvent.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    public MouseWheelMovedCanvasEvent(MouseWheelEvent mouseWheelEvent) {
+        this.mouseWheelEvent = mouseWheelEvent;
+    }
+
+    public EventListener[] getListeners() {
+        return eventListenerList.listeners;
+    }
+
```
Comment (alexanderhenne) [inline src/main/java/com/wingman/client/api/events/mouse/MouseWheelMovedCanvasEvent.java]: Please remove this extra newline

**Review by alexanderhenne — CHANGES_REQUESTED** [review]

**Review by mattthw — COMMENTED** [review]
changes made

**Pull request merged**